### DISCUSSION TITLE
Autocomplete `Literal` in `==` expressions

### DISCUSSION THREAD

**Discussion by not-my-profile** [microsoft/pylance-release#2635] [Enhancement]
Within arguments pyright provides autocompletion:

![2022-02-20-085343_510x198_scrot](https://user-images.githubusercontent.com/73739153/154833643-79ba75ca-cce7-4cf0-86fb-6e0ee6a9b0fa.png)

It would be nice if pyright could provide the same autocompletion within `==` expressions:

![2022-02-20-085503_334x301_scrot](https://user-images.githubusercontent.com/73739153/154833689-026ec651-39fa-4362-a03f-541e0f0303cf.png)

**Comment by erictraut** [comment]
This isn't a core type checking feature, so I'm transferring the enhancement request to the pylance-release project for consideration.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
